ssp

```diff
@@ -2045,16 +2045,18 @@ sub print_ea3_php_configuration {
         }
 
         if ( $EA3_PHP4HANDLER && $EA3_PHP4HANDLER ne 'none' ) {
-            my @php_v = split /\n/, timed_run( 0, '/usr/local/php4/bin/php', '-v' );
-            if ( @php_v && $php_v[0] =~ /^PHP\s(\S+)\s(\S+)/ ) {
-                $EA3_PHP4VERSION = $1;
-            }
-            else {
-                $EA3_PHP4VERSION = '(version unknown)';
-            }
+            if ( -x '/usr/local/php4/bin/php' ) {
+                my @php_v = split /\n/, timed_run( 0, '/usr/local/php4/bin/php', '-v' );
+                if ( @php_v && $php_v[0] =~ /^PHP\s(\S+)\s(\S+)/ ) {
+                    $EA3_PHP4VERSION = $1;
+                }
+                else {
+                    $EA3_PHP4VERSION = '(ea3 php4 version unknown)';
```

Comment:
This is not the right place to mention "ea3".  If EA3 is going to be mentioned, it should be added in the print_warn text in exactly the same way that EA4 is mentioned.  If we're going to do that then every output dealing with EA3 needs to be changed.